### PR TITLE
Update description of user privileges on UTL_FILE

### PR DESCRIPTION
**Pull request by dwicinas**
Update description now that users can be granted privileges on directories.

Addresses https://enterprisedb.atlassian.net/browse/DOCS-782

### CODE REVIEW

File: product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx [hunk 1]
```diff
@@ -15,7 +15,19 @@ The `UTL_FILE` package reads from and writes to files on the operating system’
 GRANT EXECUTE ON PACKAGE SYS.UTL_FILE TO mary;
 ```
 
-Also, the operating system username `enterprisedb` must have the appropriate read/write permissions on the directories and files that the `UTL_FILE` functions and procedures access. If the required file permissions aren't in place, an exception is thrown in the `UTL_FILE` function or procedure.
+Also, for a UTL_FILE function or procedure to process successfully, the following conditions must be true:
```
Comment (ebgitelman) [inline product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx:18]:
```suggestion
Also, for a `UTL_FILE` function or procedure to process successfully, you need:
```

File: product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx [hunk 2]
```diff
@@ -15,7 +15,19 @@ The `UTL_FILE` package reads from and writes to files on the operating system’
...
+
+- The user must have the `EXECUTE` privilege on the `UTL_FILE` package. Without that privilege the user is not able to invoke any package APIs.
```
[the remark this answers is not in the document]
Comment (ebgitelman) [reply product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx:20]: FYI, better to state in the positive than the negative. Easier to follow.

**Review by nidhibhammar — APPROVED** [review]
LGTM

**Comment by josh-heyer** [comment]
⚠Rebased - history has changed, please reset your local branch before continuing work.

File: product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_sql/64_grant.mdx [hunk 3]
```diff
@@ -35,6 +35,10 @@ GRANT { { INSERT | UPDATE | REFERENCES } (column [, ...]) }
   TO { username | groupname | PUBLIC } [, ...]
   [ WITH GRANT OPTION ]
 
+GRANT { { CREATE | DROP | READ | WRITE } }
```
Comment (ajit-at-edb) [inline product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_sql/64_grant.mdx:38]: [ WITH GRANT OPTION ] needs to be added

Comment (dwicinas) [reply product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_sql/64_grant.mdx:38]: Added, as suggested

**Comment by dwicinas** [comment]
@josh-heyer Sorry, but what does it mean to "reset your local branch." Besides fetching.

**Comment by josh-heyer** [comment]
Fetching will probably work, @dwicinas. If it doesn't (you get conflicts), delete the local branch and re-pull.

**Review by ajit-at-edb — APPROVED** [review]

**Pull request merged**